cluster/docker-cluster/config-network.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to configure & start flanneld container on bootstrap daemon
+# and then configure & restart docker daemon to make network parameters work.
+
+# Users are allowed to re-wirte this script to deploy their own network solution
+# instead of flannel. 
+
+set -e
+
+source ~/docker-cluster/kube-config/node.env
+
+if [ "$(id -u)" != "0" ]; then
+  echo >&2 "config-network.sh must be called as root!"
+  exit 1
+fi
+
+# Only need to do this on master, so we use MASTER_IP in $1 as a flag
+if [[ ! -z $1 ]]; then
+  docker -H unix:///var/run/docker-bootstrap.sock run \
+    --net=host gcr.io/google_containers/etcd:$ETCD_VERSION \
+    etcdctl set /coreos.com/network/config \
+    "{ \"Network\": \"$FLANNEL_NET\", \"Backend\": {\"Type\": \"vxlan\"}}"
```

Comment:
Sorry, this case flannel will complain FLANNEL_NET invalid. So better not to eliminate escaping.